src/deadline/job_attachments/README.md

```diff
@@ -56,7 +56,7 @@ This content-addressable approach ensures that identical files are only stored o
 
 #### Manifest Files
 
-Manifests are stored for both job inputs and task outputs. Input manifests for jobs are stored under:
+Manifests are stored for both job inputs and task outputs. The S3 key for the input manifest for a job can be found by [calling GetJob](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_GetJob.html#API_GetJob_ResponseSyntax) and looking in the response under `attachments.manifests.rootPath`. While the input manifest location is not formally defined, in the current job attachments implementation it follows the pattern:
```

Comment:
>While the input manifest location is not formally defined

It is formally defined to be the value returned from that API. Maybe say "While the input manifest does not have a requirement on its pattern, ..."